test/unit/gateways/migs_test.rb

```diff
@@ -5,16 +5,20 @@ def setup
     @gateway = MigsGateway.new(
                  :login => 'login',
                  :password => 'password',
-                 :secure_hash => '76AF3392002D202A60D0AB5F9D81653C'
+                 :secure_hash => '76AF3392002D202A60D0AB5F9D81653C',
+                 advanced_login: 'advlogin',
+                 advanced_password: 'advpass'
```

Comment:
I'm not sure if Rubocop would catch this, but it's idiomatic to keep the same kind of notation here (either `asdf: 'qwer'` or :`asdf => 'qwer'` but not both.

Reply:
Thanks for catching that! I copied those from another part of the code. Is there any preference between keeping the hash rockets or changing over to the more modern syntax?

Reply:
Probably prefer modern over rockets, but nothing concrete.

Reply:
Changed all of the unit test hash rockets to the modern syntax. ✅